core/src/test/scala/unit/kafka/server/ReplicaManagerTest.scala

```diff
@@ -3701,6 +3721,11 @@ class ReplicaManagerTest {
 
       assertEquals(None, replicaManager.replicaFetcherManager.getFetcher(topicPartition))
 
+      if (enableRemoteStorage) {
```

Comment:
Do you also want to add similar coverage for both the below tests?

```
testFetcherAreNotRestartedIfLeaderEpochIsNotBumpedWithKRaftPath
testReplicasAreStoppedWhileInControlledShutdownWithKRaft
```

Reply:
Good catch! I thought I've covered every KRaft test cases. Updated.